pkg/oc/cli/cmd/create/clusterquota.go

```diff
@@ -35,79 +32,87 @@ var (
 )
 
 type CreateClusterQuotaOptions struct {
-	ClusterQuota *quotaapi.ClusterResourceQuota
-	Client       quotaclient.ClusterResourceQuotasGetter
+	CreateSubcommandOptions *CreateSubcommandOptions
 
-	DryRun bool
+	LabelSelectorStr      string
+	AnnotationSelectorStr string
+	Hard                  []string
 
-	OutputFormat string
-	Out          io.Writer
-	Printer      ObjectPrinter
-}
+	ProjectLabelSelectorStr      *metav1.LabelSelector
+	ProjectAnnotationSelectorStr map[string]string
 
-type ObjectPrinter func(runtime.Object, io.Writer) error
+	Client quotav1client.ClusterResourceQuotasGetter
+}
 
 // NewCmdCreateClusterQuota is a macro command to create a new cluster quota.
-func NewCmdCreateClusterQuota(name, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
-	o := &CreateClusterQuotaOptions{Out: out}
-
+func NewCmdCreateClusterQuota(name, fullName string, f genericclioptions.RESTClientGetter, streams genericclioptions.IOStreams) *cobra.Command {
+	o := &CreateClusterQuotaOptions{
```

Comment:
I didn't do it before, but let's create `NewCreateClusterQuotaOptions` method like we do in all other places. It's nice and clean solution.